### PR TITLE
add celestia team

### PR DESCRIPTION
**Pull request by galargh**
### Summary
Celestia is contributing to testground/infra. They should become testground members.

### Why do you need this?
To make it easier to contribute.

### What else do we need to know?
n/a

**DRI:** myself

### Reviewer's Checklist

- [ ] It is clear where the request is coming from (if unsure, ask)
- [ ] All the automated checks passed
- [ ] The YAML changes reflect the summary of the request
- [ ] The Terraform plan posted as a comment reflects the summary of the request

### CODE REVIEW

File: github/testground.yml [hunk 1]
```diff
@@ -242,6 +244,12 @@ teams:
         - StefanGajic
         - StefanMiletich
     privacy: closed
+  celestia:
+    description: https://celestia.space/
+    members:
+      - Bidon15
+      - sysrex
```
Comment (galargh) [inline github/testground.yml:251]:
```suggestion
    members:
      member:
        - Bidon15
        - sysrex
```

File: github/testground.yml [hunk 2]
```diff
@@ -72,6 +74,8 @@ repositories:
       maintain:
         - bloxico
         - maintainers
+      push:
+        - celestia
```
Comment (galargh) [inline github/testground.yml:78]: With GitHub Management in place I think `push` should be our default. @laurentsenta, can you think of any actions that might require higher permissions than that?

Comment (laurentsenta) [reply github/testground.yml:78]: I think `push` make sense,
we might want to give auth on `testground/testground` repo also; that's where issues are [centralized](https://github.com/testground/infra/#contribute).

Comment (laurentsenta) [reply github/testground.yml:78]: (thanks for the update)

**Review by laurentsenta — APPROVED** [review]
Boom! Welcome @Bidon15 and @sysrex

**Pull request merged**